web/ui/dashboard/src/app/private/components/create-project-component/create-project-component.component.ts

```diff
@@ -123,6 +123,8 @@ export class CreateProjectComponent implements OnInit {
 		try {
 			const projectDetails = this.privateService.getProjectDetails;
 
+			this.projectDetails = projectDetails;
```

Comment:
Do we need to assign two variables here?

We have the first `projectDetails` local variable and we're assigning it to another global variable.

Check how we're using these two variables, I'm sure one can be used in place of the other.